pkgs/development/libraries/mesa/default.nix

```diff
@@ -214,9 +218,10 @@ self = stdenv.mkDerivation {
     # Clover, old OpenCL frontend
     "-Dgallium-opencl=icd"
     "-Dopencl-spirv=true"
-
     # Rusticl, new OpenCL frontend
-    "-Dgallium-rusticl=true" "-Drust_std=2021"
```

Comment:
Rusticl always needs-rust

Reply:
It's logically impossible to have `enableOpenCL` without `needsRust` here.